force-app/main/default/classes/CarePgmEnrolleeTriggerHandler.cls

```diff
@@ -32,4 +34,27 @@ public with sharing class CarePgmEnrolleeTriggerHandler {
             }
         }        
     }
+    
+/**
+* @author: Deepak
+* @date: 11/07/2024
+* @description: The purpose of this method is update the Account Ref. No based on CPE Name
+* @Modification Log: [Date] - [Change Reference] - [Changed By] - [Description]
+*/ 
+    public static void updateEnrolleeNameOnAcc(List<CareProgramEnrollee> carePgmEnrolleeList){
+        List<Account> accList = new List<Account>();
+        String cPEDRD = 'EDRD';
+        List<CareProgramEnrollee> cPEList =[SELECT Id, CareProgram.Name FROM CareProgramEnrollee 
```

Comment:
This should be part of selector class

Reply:
@yashrohan-cgi , we do not have a frame setup for selector class.